nextcloudappstore/scaffolding/views.py

```diff
@@ -25,6 +28,10 @@ def get_initial(self):
         return init
 
     def form_valid(self, form):
+        if settings.APP_SCAFFOLDING_LOG:
+            with open(settings.APP_SCAFFOLDING_LOG, 'a', newline='') as csvfile:
```

Comment:
Since this contains personal information, a notice should be added in the webpage.

Aside from that code looks good 👍

Reply:
Yep, good point.